Formula/docker2aci.rb

```diff
@@ -0,0 +1,28 @@
+class Docker2aci < Formula
+  desc "Library and CLI tool to convert Docker images to ACIs"
+  homepage "https://github.com/appc/docker2aci"
+  url "https://github.com/appc/docker2aci/archive/v0.16.0.tar.gz"
+  sha256 "7a8abe8b809a0dcf7a583c6a8ba8232eedb97d30750fa4b4f0620e9db6e504fb"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOOS"] = "darwin"
+    ENV["GOARCH"] = MacOS.prefer_64_bit? ? "amd64" : "386"
```

Comment:
You probably don’t need to set these two variables since we’re not doing cross-compilation here.